lxd/instance/drivers/driver_qemu.go

```diff
@@ -1313,6 +1313,136 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 		qemuCmd = append(qemuCmd, "-smbios", "type=2,manufacturer=Canonical Ltd.,product=LXD")
 	}
 
+	// If user has requested AMD SEV, check if supported and add to QEMU command.
+	if shared.IsTrue(d.localConfig["security.sev"]) {
```

Comment:
Lets move this whole bit into a separate function called from here as `start()` is already too long IMHO.
Its getting hard to follow what the function is doing when you're not concerned with the nitty gritty detail of each step.